lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -32,6 +35,7 @@ def activate_shopify_session
         ShopifyAPI::Context.activate_session(current_shopify_session)
         yield
       ensure
+        ShopifyApp::Logger.debug("Deactivating Session")
```

Comment:
How come for some logs we capitalize each word like it's a title? Is this a format that we're doing in the node library as well /cc @paulomarg

Reply:
No, we've used sentence case pretty much everywhere for node!

Reply:
ok just went through and tried to clean that up